src/ng/sniffer.js

```diff
@@ -29,7 +29,8 @@ function $SnifferProvider() {
             $window.chrome &&
             ($window.chrome.app && $window.chrome.app.runtime ||
                 !$window.chrome.app && $window.chrome.runtime && $window.chrome.runtime.id),
-        hasHistoryPushState = !isChromePackagedApp && $window.history && $window.history.pushState,
+        isNw = $window.nw && $window.nw.process,
```

Comment:
Since `isNw` is actually part of detecting if it `isChromePackagedApp`, it is clearer to incorporate it in there:

```js
isNw = ...;
isChromePackagedApp = !isNw && ...;
```